pkg/controllers/cloud/account_poller.go

```diff
@@ -141,26 +144,33 @@ func (p *Poller) addAccountPoller(cloudType cloudv1alpha1.CloudProvider, namespa
 	return poller, false
 }
 
-// removeAccountPoller removes an account poller from accPollers map.
+// removeAccountPoller stops go-routine and removes an account poller from accPollers map.
 func (p *Poller) removeAccountPoller(namespacedName *types.NamespacedName) error {
 	p.mutex.Lock()
 	defer p.mutex.Unlock()
 
-	poller, found := p.accPollers[*namespacedName]
-	if found {
-		if poller.selector != nil {
-			cloudInterface, err := cloudprovider.GetCloudInterface(common.ProviderType(poller.cloudType))
-			if err != nil {
-				return err
+	if poller, found := p.accPollers[*namespacedName]; found {
```

Comment:
whats the difference between this poller and poller object (line 145). Can we come up with better naming for these two?

Reply:
Account poller seems appropriate to me. Probably "Poller" needs to change, Do you have any recommendations for "Poller" ?